util.c

```diff
@@ -787,3 +787,65 @@ char *filter_printf(const char *filter, const char *user) {
   filter_result_len(filter, user, result);
   return result;
 }
+
+size_t rfc4515_length ( const char* in, char* out )
+{
+    const char* pos = NULL;
+    size_t result = 0;
+    char* specials = (char*)malloc( 4 * sizeof(char) );
```

Comment:
Since this is data known at compile time, create it as a const string instead. It's important to note that the strpbrk() call later will read it as a string, so it has to be NULL terminated.

Reply:
Hey @klali, thank you for your note. I completely forgot to terminate the string. In my first version I created `specials` as `const char*`. But I changed this because strpbrk(...) did not accept a `const char *` and produced an compiler error. What du you think about that? Is this a problem for you?